src/main/java/com/internet/shop/web/filter/AuthorisationFilter.java

```diff
@@ -0,0 +1,72 @@
+package com.internet.shop.web.filter;
+
+import com.internet.shop.controller.LoginController;
+import com.internet.shop.lib.Injector;
+import com.internet.shop.model.Role;
+import com.internet.shop.model.User;
+import com.internet.shop.service.UserService;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.annotation.WebFilter;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebFilter("/*")
+public class AuthorisationFilter implements Filter {
```

Comment:
```suggestion
public class AuthorizationFilter implements Filter {
```
also, can you guarantee the order in which your filters execute? let's indicate order in web.xml

Reply:
done